daffodil-codegen-c/src/main/scala/org/apache/daffodil/codegen/c/generators/CodeGeneratorState.scala

```diff
@@ -400,7 +427,7 @@ class CodeGeneratorState(private val root: ElementBase) {
       if (numChildren > 0)
         s"""static const $C ${C}_compute_offsets;
          |
-         |static const size_t ${C}_offsets[$count] = {
+         |static const size_t ${C}_childrenOffsets[$count] = {
```

Comment:
Yes, I didn't want to do a comprehensive job of renaming in this PR.  The remaining camel case identifiers are in "types", which include struct field names as well, and it sort of makes sense to keep them closer to Daffodil's camel case style since both the Daffodil and C types talk about similar parsing/unparsing/metadata data.